.prettierrc

```diff
@@ -1,5 +1,6 @@
 {
   "singleQuote": true,
+  "trailingComma": "es5",
```

Comment:
The default trailingCommas rule has been changed to `all` [in v3](https://prettier.io/docs/en/options.html#trailing-commas). It would have introduced +3000~ instances to change, so I've chosen to update the config with the previous default option, `es5`.